actors.md

```diff
@@ -477,31 +461,53 @@ type StorageMinerActorState struct {
     dePledgeTime BlockHeight
 
 	## All sectors this miner has committed.
-    sectors SectorSet
+    sectors &SectorSet
 
 	## Sectors this miner is currently mining. It is only updated
 	## when a PoSt is submitted (not as each new sector commitment is added).
-    provingSet SectorSet
+    provingSet &SectorSet
 
 	## Sectors reported during the last PoSt submission as being 'done'. The collateral
     ## for them is still being held until the next PoSt submission in case early sector
     ## removal penalization is needed.
     nextDoneSet BitField
 
 	## Deals this miner has been slashed for since the last post submission.
-    arbitratedDeals CidSet
+    arbitratedDeals {Cid:Null}
 
 	## Amount of power this miner has.
     power UInt
 
     ## List of sectors that this miner was slashed for.
-    slashedSet optional SectorSet
+    slashedSet optional &SectorSet
```

Comment:
If the provingSet is a BitField, would this one be too?

Reply:
This one will have to be an actual sector set I think. otherwise it will be hard to keep track of which sectors are which, i think.. cc @dignifiedquire ?

Reply:
hmmmmmmmmm I am not sure, lets leave it as is in this PR, and I'll think about it more deeply in my cleanup work